src/current/molt/molt-fetch.md

```diff
@@ -369,10 +372,106 @@ To cancel replication, enter `ctrl-c` to issue a `SIGTERM` signal. This returns
+
+{{site.data.alerts.callout_info}}
+If there is already a running CockroachDB changefeed with the same webhook sink URL (excluding query parameters) and [watched rows]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}), the existing changefeed is used for `failback`.
+{{site.data.alerts.end}}
+
+By default, `failback` uses an insecure configuration for the CockroachDB changefeed. The [default configuration](#default-insecure-changefeed) is suited for testing only. In production, you should override the default changefeed settings with a **secure** configuration. To override the default changefeed settings, create a JSON file and include the [`--changefeeds-path` flag](#global-flags) to indicate the file path. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+--changefeeds-path 'changefeed-settings.json'
+~~~
+
+You can specify the following [`CREATE CHANGEFEED` parameters]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#parameters) in the override JSON. If any parameter is not specified, its [default value](#default-insecure-changefeed) is used.
+
+- The following [`CREATE CHANGEFEED` URI parameters]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#sink-uri):
+	- `host`: The hostname or IP address of the [webhook sink]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}#webhook-sink) where change events are sent. The applicable certificates of the failback target (i.e., the [source database](#source-and-target-databases) from which you migrated) **must** be located on this machine.
+	- `port`: The port of the [webhook sink]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}#webhook-sink).
+	- `sink_query_parameters`: A comma-separated list of [`CREATE CHANGEFEED` query parameters]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#query-parameters). This includes the base64-encoded client certificate ([`client_cert`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#client-cert)), key ([`client_key`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#client-key)), and CA ([`ca_cert`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#ca-cert)) for a secure webhook sink.
+- The following [`CREATE CHANGEFEED` options]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#options):
+	- [`resolved`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#resolved)
+	- [`min_checkpoint_frequency`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#min-checkpoint-frequency)
+	- [`initial_scan`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#initial-scan)
+	- [`webhook_sink_config`]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#webhook-sink-config)  
+
+To secure the changefeed connection, define `sink_query_parameters` in the JSON as follows:
```

Comment:
Yeah, I think this should be moved up above the insecure. Insecure should be noted but shouldn't be the focal point. We should show them examples of how to do secure first and mention: if you know what you're doing and need to run insecure and with default configs, here is what you need to do.

Reply:
Done. Let me know if it's still not what you're looking for.

Reply:
Excellent. This is exactly what I wanted. One minor note here is maybe we should emphasize that insecure configurations are not recommended and to run in secure mode wherever possible. Like explicitly bold it so it's clear and that they take on any risk for insecure modes.